### PR TITLE
zh-translation: /docs/concepts/security/index.md

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1656

### CODE REVIEW

File: content/zh/docs/concepts/security/index.md [hunk 1]
```diff
@@ -400,62 +337,56 @@ peers:
...
+- **什么** 请参考 `AuthorizationPolicy` 的 `rule` 中的 `to` 部分。
+- **条件** 请参考 `AuthorizationPolicy` 的 `rule` 中的 `when` 部分。
 
-Each rule has the following standard fields:
+每个规则都有以下标准字段：
 
-- **`from`**: A list of sources.
-- **`to`**: A list of operations.
-- **`when`**: A list of custom conditions.
+- **`from`**: 来源列表。
```
Comment (yuxiaobo96) [inline content/zh/docs/concepts/security/index.md]:
```suggestion
- **`from`**：来源列表。
```

File: content/zh/docs/concepts/security/index.md [hunk 2]
```diff
@@ -482,23 +413,19 @@ spec:
...
+#### 策略目标{#policy-target}
 
-Policy scope (target) is determined by `metadata/namespace` and an optional `selector`.
+策略范围（目标）由 `metadata/namespace` 和可选的 `selector` 确定。
 
-The `metadata/namespace` tells which namespace the policy applies to. If set to the
-root namespace, the policy applies to all namespaces in a mesh. The value of
-root namespace is configurable, and the default is `istio-system`. If set to a
-normal namespace, the policy will only apply to the specified namespace.
+`metadata/namespace` 告诉该策略适用于哪个命名空间。如果设置为根名称空间，则该策略将应用于网格中的所有命名空间。根命名空间的值是可配置的，默认值为 `istio-system`。
```
Comment (yuxiaobo96) [inline content/zh/docs/concepts/security/index.md]:
```suggestion
`metadata/namespace` 告诉该策略适用于哪个命名空间。如果设置为根命名空间，则该策略将应用于网格中的所有命名空间。根命名空间的值是可配置的，默认值为 `istio-system`。
```

File: content/zh/docs/concepts/security/index.md [hunk 3]
```diff
@@ -400,62 +337,56 @@ peers:
...
+- **条件** 请参考 `AuthorizationPolicy` 的 `rule` 中的 `when` 部分。
 
-Each rule has the following standard fields:
+每个规则都有以下标准字段：
 
-- **`from`**: A list of sources.
-- **`to`**: A list of operations.
-- **`when`**: A list of custom conditions.
+- **`from`**: 来源列表。
+- **`to`**: 操作列表。
```
Comment (yuxiaobo96) [inline content/zh/docs/concepts/security/index.md]: 同上

File: content/zh/docs/concepts/security/index.md [hunk 4]
```diff
@@ -400,62 +337,56 @@ peers:
...
+- **`when`**: 自定义条件列表。
```
Comment (yuxiaobo96) [inline content/zh/docs/concepts/security/index.md]: 同上

File: content/zh/docs/concepts/security/index.md [hunk 5]
```diff
@@ -516,19 +443,17 @@ spec:
...
-of the field with a few exceptions (e.g., the `key` field under the `when` section,
-the `ipBlocks` under the `source` section and the `ports` field under the `to` section only support exact match).
+大部分字段都支持完全匹配、前缀匹配、后缀匹配和存在匹配，但有一些例外情况（例如，`when` 部分下的`key` 字段，`source` 部分下的 `ipBlocks` 和 `to` 部分下的 `ports` 字段仅支持完全匹配）。
 
-- **Exact match**. i.e., exact string match.
-- **Prefix match**. A string with an ending `"*"`. For example, `"test.abc.*"` matches `"test.abc.com"`, `"test.abc.com.cn"`, `"test.abc.org"`, etc.
-- **Suffix match**. A string with a starting `"*"`. For example, `"*.abc.com"` matches `"eng.abc.com"`, `"test.eng.abc.com"`, etc.
-- **Presence match**. `*` is used to specify anything but not empty. You can specify a field must be present using the format `fieldname: ["*"]`.
-This means that the field can match any value, but it cannot be empty. Note that it is different from leaving a field unspecified, which means anything including empty.
+- **完全匹配**。 即完整的字符串匹配。
```
Comment (yuxiaobo96) [inline content/zh/docs/concepts/security/index.md]:
```suggestion
- **完全匹配**。即完整的字符串匹配。
```
下同

Comment (ilylia) [reply content/zh/docs/concepts/security/index.md]: all fixed

**Comment by yuxiaobo96** [comment]
@rootsongjc 
已修改，可以合并

**Pull request merged**